scripts/ci/install_deps.sh

```diff
@@ -21,6 +21,12 @@ if [[ -n "$REQUIRES_MINIKUBE" ]]; then
   sudo chown -R travis: /home/travis/.minikube/
   kubectl cluster-info
 
+  # In some tests (specifically TestKubernetesTLSRotation), we observe incorrect DNS resolution 
+  # after pods have been re-created which causes problems with inter pod communicataion.
+  # Set CoreDNS TTL to 0 so that DNS entries are not cached. 
+  kubectl get cm coredns -n kube-system -o yaml | sed -e 's/ttl [0-9]*$/ttl 0/' | kubectl apply -n kube-system -f -
```

Comment:
Have looked at `kubectl patch`? This would be more structured and less brittle.

Reply:
@adriansuarez I normally use `kubectl patch` when there is a need to change something specific in K8s resource spec. AFAIK with configmaps, you can patch the `data: {config: "MULTILINE TEXT"}` and not specifically the `ttl` property in the text. I didn't want to overwrite the full CoreDNS config as it might change in future.